src/style.css

```diff
@@ -1 +1,68 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
for better performance, it is recommended to import font in HTML files

Reply:
ok, I will do it. tests have passed 2 out of  8 now I am fixing it.